src/engine/video/image.cpp

```diff
@@ -326,10 +320,12 @@ bool ImageDescriptor::SaveMultiImage(const std::vector<StillImage *>& images, co
     float img_height = images[0]->_height;
     for(uint32_t i = 0; i < images.size(); i++) {
         if(images[i] == nullptr || images[i]->_image_texture == nullptr) {
-            IF_PRINT_WARNING(VIDEO_DEBUG) << "nullptr StillImage or ImageElement was present in images vector argument when saving file: " << filename << std::endl;
+            IF_PRINT_WARNING(VIDEO_DEBUG) << "nullptr StillImage or ImageElement was present in images vector argument when saving file: "
+                                          << filename << std::endl;
             return false;
         }
-        if(IsFloatEqual(images[i]->_width, img_width) == false || IsFloatEqual(images[i]->_height, img_height)) {
+        if(!IsFloatEqual(images[i]->_width, img_width) ||
+           IsFloatEqual(images[i]->_height, img_height)) {
```

Comment:
I think there was a bug here. Actually, the code should be:
`if(!IsFloatEqual(images[i]->_width, img_width) || !IsFloatEqual(images[i]->_height, img_height))`
Could you fix this?